services/app/src/hooks/useChessBoard.js

```diff
@@ -0,0 +1,27 @@
+import { useState } from 'react';
+import { numberOfRows, numberOfColumns } from '../constants/board';
+
+const alphabetArray = [...'abcdefghijklmnopqrstuvwxyz'];
+
+export const useChessBoard = () => {
+  const [board, setBoard] = useState(
+    [...Array(numberOfRows + 1).keys()].map((_row, rowIndex) =>
+      [...Array(numberOfColumns + 1).keys()].map((_tile, tileIndex) => ({
```

Comment:
Python has a `range` function, you could make yourself something similar here, a function that returns an array from x to y